setup.py

```diff
@@ -21,37 +20,7 @@
     exec(version_file.read(), VERSION)
 
 setup(
-    name="scispacy",
     version=VERSION["VERSION"],
-    url="https://allenai.github.io/scispacy/",
-    author="Allen Institute for Artificial Intelligence",
-    author_email="ai2-info@allenai.org",
-    description="A full SpaCy pipeline and models for scientific/biomedical documents.",
-    long_description=open("README.md").read(),
-    long_description_content_type="text/markdown",
-    keywords=["bioinformatics nlp spacy SpaCy biomedical"],
-    classifiers=[
-        "Intended Audience :: Science/Research",
-        "Development Status :: 3 - Alpha",
-        "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python :: 3.6",
-        "Topic :: Scientific/Engineering :: Artificial Intelligence",
-        "Topic :: Scientific/Engineering :: Bio-Informatics",
-    ],
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
```

Comment:
i'm not sure whether this is still necessary or not (you can remove it, install with toml, and see if the test files are included or not). if it is, i believe the modern version is

```
[tool.setuptools.packages.find]
exclude = ["*.tests", "*.tests.*", "tests.*", "tests"]
```

Reply:
Done in 9cb79a3